tlcc_enclave/enclave/ecc_json.cpp

```diff
@@ -8,6 +8,25 @@
 #include "base64.h"
 #include "parson.h"
 
+int marshal_ecc_args(std::vector<std::string>& argss,
+    std::string& encoded_string)  // assumed to be initialized but empty
+{
+    JSON_Value* root = json_value_init_array();
+    JSON_Array* list = json_value_get_array(root);
+
+    for (std::size_t i = 0; i < argss.size(); i++)
+    {
+        json_array_append_string(list, argss[i].c_str());
+    }
+
+    char* ss = json_serialize_to_string(root);
+    encoded_string.append(ss);
+
+    json_free_serialized_string(ss);
+    json_value_free(root);
+    return 1;
```

Comment:
we always return 1 here. is there also a chance that parson returns an error we might want to return here? otherwise marshal_ecc_args may also return void?!

Reply:
I followed the precedent of the other already existing function which also doesn't do error checking.  If i would have had more time and the PR wouldn't have been already that big i might have changed it already here but now it has to wait for another occasion  ...